root/static/scripts/common.js

```diff
@@ -33,6 +33,7 @@ window.ko = require('knockout');
 window.$ = window.jQuery = require('jquery');
 
 require('../lib/jquery.ui/ui/jquery-ui.custom');
+require('../lib/bootstrap.min');
```

Comment:
Selenium tests are failing because this is conflicting with jQuery; the following error is logged when clicking on the "Add Medium" button in the release editor.

```
bootstrap.min.js:6 Uncaught TypeError: No method named "option"
    at HTMLAnchorElement.<anonymous> (bootstrap.min.js:6:49635)
    at Function.each (jquery.js:384:1)
    at jQuery.fn.init.each (jquery.js:136:1)
    at jQuery.fn.init.jQueryInterface [as tooltip] (bootstrap.min.js:6:49533)
    at init.js:181:14
    at utils.js:39:21
    at evaluateImmediate (index.js:1638:1)
    at ko.subscription.evaluatePossiblyAsync [as callback] (index.js:1581:1)
    at Function.notifySubscribers (index.js:1063:1)
    at Function.observable.valueHasMutated (index.js:1241:48)
```